src/Layers/xrRender/r_constants.h

```diff
@@ -101,14 +105,17 @@ struct ECORE_API R_constant : public xr_resource
 
     R_constant_load ps;
     R_constant_load vs;
-#ifndef USE_DX9
-    R_constant_load gs;
-#ifdef USE_DX11
+
+#if defined(USE_DX11) || defined(USE_OGL)
+	R_constant_load gs; //ogl OR dx11
+#endif
+
+#if defined(USE_DX11)
     R_constant_load hs;
     R_constant_load ds;
     R_constant_load cs;
 #endif
```

Comment:
- ps, vs and others should be near each other, they are related. Thus, whitespace is not needed.
- GS (Geometry Shaders) exist on many graphics APIs, so "if not exist" check is shorter and we won't need to add each new API to this check.
```suggestion
#ifndef USE_DX9
    R_constant_load gs;
#   if defined(USE_DX11)
    R_constant_load hs;
    R_constant_load ds;
    R_constant_load cs;
#   endif
#endif
```